com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -85,6 +85,8 @@ public enum ShaderVariantLogLevel
         Disabled,
         OnlyURPShaders,
         AllShaders,
+        [Obsolete("OnlyUniversalRPShaders is deprecated, use OnlyURPShaders instead (UnityUpgradable) -> OnlyURPShaders", true)]
+        OnlyUniversalRPShaders,
```

Comment:
OnlyURPShaders and OnlyUniversalRPShaders have different int values. Will that cause any issue with upgrading?
 Actually, do we need to change this? Is this change driven by how it appears on UI? Maybe if that's the case we could change the UI name only?

Reply:
I've checked, and the enums are stored with their underlaying type, so an int,as the value of the renamed enum is the same as before the serialization is working as expected, and the UnityUpgradable is checking code string so it was working fine, but maybe an user can be doing something that we are not expecting, so I decided to go on your approach and only change the way that we display the enum

Added an inspector name attribute to have:

![image](https://user-images.githubusercontent.com/76204843/127460656-6bc88784-277e-44d4-a1da-09d2fc355514.png)